site/content/en/docs/Guides/Client SDKs/csharp.md

```diff
@@ -25,10 +33,14 @@ Download the source {{< ghlink href="sdks/csharp" >}}directly from GitHub{{< /gh
 
 Reference the SDK in your project & create a new instance of the SDK wrapper:
 
+### Initialization

```

Comment:
Might be a good time to ask - will there be any kind of `import` statement that needs to happen at the top of the file -- might be good to include that here?

Reply:
Yeah, will get it updated 👌